packages/kit/types/index.d.ts

```diff
@@ -83,12 +84,13 @@ export interface Builder {
 	config: ValidatedConfig;
 	/** Information about prerendered pages and assets, if any. */
 	prerendered: Prerendered;
-	/** `true` if one or more routes have `export const config = ..` in it */
-	hasRouteLevelConfig: boolean;
+	/** An array of dynamic (not prerendered) routes */
+	routes: RouteDefinition[];
```

Comment:
Note for later: The Vercel adapter needs a major version bump because the peer dependency on kit needs a bump so it can expect this API to be present.

Reply:
Adapter-Auto, too, then. So we also need to update `create-svelte` to use the latest version.